src/device.rs

```diff
@@ -253,10 +261,10 @@ pub unsafe extern "C" fn wgpu_adapter_request_device(
     gfx_select!(adapter_id => GLOBAL.adapter_request_device(adapter_id, &desc, trace_path, PhantomData))
 }
 
-#[no_mangle]
-pub extern "C" fn wgpu_adapter_extensions(adapter_id: id::AdapterId) -> wgt::Extensions {
-    gfx_select!(adapter_id => GLOBAL.adapter_extensions(adapter_id))
-}
+// #[no_mangle]
+// pub extern "C" fn wgpu_adapter_extensions(adapter_id: id::AdapterId) -> wgt::Extensions {
+//     gfx_select!(adapter_id => GLOBAL.adapter_extensions(adapter_id))
+// }
```

Comment:
I couldn't find a replacement for this

Reply:
The term "extensions" has been replaced with "features"